javascript-algorithms-and-data-structures/pokemon-search-app/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pokémon Search App</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h1>Pokémon Search App</h1>
+    <div class="container">
+      <form role="search" id="search-form">
+        <label for="search-input" class="sr-only"
+          >Search for Pokémon Name or ID</label
```

Comment:
Right now, I am able to search by name

<img width="511" alt="Screenshot 2023-08-30 at 9 48 26 AM" src="https://github.com/freeCodeCamp/CurriculumExpansion/assets/67210629/df1f2ab1-0677-47e2-a618-11f7e13471c5">


But I am not able to search by id
<img width="590" alt="Screenshot 2023-08-30 at 9 49 30 AM" src="https://github.com/freeCodeCamp/CurriculumExpansion/assets/67210629/8f4551c1-c602-4269-a6a8-5419276f6547">

I think we should either update the JS to include being able to search by id

Or we should update the messaging here to only search by name and drop the ID completely

Reply:
Hi Jessica, the ID is searchable without the `#`, like this:

![image](https://github.com/freeCodeCamp/CurriculumExpansion/assets/33151350/412135cd-586c-44c8-bcfc-19b9e5ba67ec)

Reply:
@zairahira 

As long as it is super clear in the user stories that user can search just by number or name then we will be good to go 👍